.editorconfig

```diff
@@ -14,6 +14,6 @@ indent_size = 4
 indent_style = space
```

Comment:
should we just get rid of the editorconfig now?

Reply:
I think it makes sense to include both, so people using editors without a prettier plugin or vscode will still have some basic styling setup.